fmt/xm.c

```diff
@@ -389,6 +389,8 @@ static int load_xm_instruments(song_t *song, struct xm_file_header *hdr, slurp_t
 	int n, ni, ns;
 	int abssamp = 1; // "real" sample
 	int32_t ihdr, shdr; // instrument/sample header size (yes these should be signed)
+	int l_start, l_end; // envelope loop nodes
```

Comment:
What's the point of these two variables?

Reply:
Oh, I see you're using `l_end` a lot as shorthand for `ins->vol_env.loop_end`, but only using `l_start` once per block. FWIW I'd rather `l_end` be a pointer to `ins->vol_env.loop_end` so that they're *actually* the same thing and don't need to be explicitly synchronized.